Build/Common.Build.Core.settings

```diff
@@ -32,12 +34,12 @@
     The version number that will identify a specific build. This appears in
     Programs and Features, Extension Manager, and Help About.
     -->
-    <FileVersionPrefix Condition="$(FileVersionPrefix) == ''">15.8</FileVersionPrefix>
+    <FileVersionPrefix Condition="$(FileVersionPrefix) == ''">$(VSTarget)</FileVersionPrefix>
```

Comment:
I've found having the minor version to be very useful (15.8 instead of 15.0). Why do you want to get rid of it?

Reply:
Ah, good point. I had a change to make it 16.0, and forgot that it previously had the minor version.